Header header
float32 temperature
uint16 col1
uint16 row1
uint16 col2
uint16 row2
